Text/TextAnalyticsForHealth/README.md

```diff
@@ -70,10 +64,63 @@ The skill also accepts options which can be sent as headers in the request.
             "data": {
                 "status": "succeeded",
                 "entities": [
-                    {...}
+                    {
+                        "text": "100mg",
+                        "category": "Dosage",
+                        "subCategory": null,
+                        "confidenceScore": 1.0,
+                        "offset": 0,
+                        "length": 5,
+                        "dataSources": [],
+                        "assertion": null,
+                        "normalizedText": null
+                    },
+                    {
+                        "text": "Ibuprofin",
```

Comment:
It's Ibuporofen, not Ibuprofin.

Reply:
I've said it before and I'll say it again, my greatest flaw is that I cannot spell things correctly. Thanks for the catch lol